EDDiscovery/EDDiscoveryForm.cs

```diff
@@ -399,12 +414,19 @@ private void GetEDSMSystems()
                     DBUpdateEDDB(eddb);
                 }
 
-                string retstr = edsm.GetNewSystems(db);
-                Invoke((MethodInvoker)delegate
+                if (CanSkipSlowUpdates())
                 {
-                    TravelHistoryControl.LogText(retstr);
-                });
-
+                    LogLine("Skipping loading updates (DEBUG option).");
+                    LogLine("  Need to turn this back on again? Look in the Settings tab.");
+                }
+                else
+                {
+                    string retstr = edsm.GetNewSystems(db);
+                    Invoke((MethodInvoker)delegate
+                    {
+                        TravelHistoryControl.LogText(retstr);
+                    });
```

Comment:
The "diffs" are a little hard to follow here, but its the `edsm.GetNewSystems(db)` part that getting optionally skipped over.